manifest.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "Betaflight - Blackbox Explorer",
   "description": "Interactive flight log viewer for Betaflight",
-  "version": "3.0.0",
-  "version_name": "3.0.0",
```

Comment:
Maybe not a good idea remove the `version_name` without modifying the `tools.js`:
https://github.com/betaflight/blackbox-log-viewer/blob/2525d26608844aa00ece7ef215a9c70b374c6d0a/js/tools.js#L466

Reply:
Check the following line.

`version_name` was a shim that I added to be able to have versions that display as `x.y.z-RCa`, but are still accepted in the chrome web store that expects version numbers to be `x.y.z`. For non-RC versions it should be omitted.

Reply:
Ah, ok. So if it is not found, it will fallback to `version`.